scripts/verify.bash

```diff
@@ -58,24 +64,18 @@ UserNotFoundf
 # actually seem to make a difference under 1.6 either don't bother.
 all_prints=`echo $logging_prints $error_prints | tr " " ,`
 
-go tool vet \
+go vet \
    -all \
    -composites=false \
    -printfuncs=$all_prints \
-    $FILES || [ -n "$IGNORE_VET_WARNINGS" ]
-
-echo "checking: copyright notices are in place ..."
-./scripts/copyright.bash
-
-echo "checking: dependency files ..."
-dep check
+    ./... || [ -n "$IGNORE_VET_WARNINGS" ]
```

Comment:
If we are doing this, is it just that it only supports a single argument? or is it that it only supports packages?
because we could do something like
DIRNAMES=$(dirname $FILES | sort -u)
Then we have the unique directories with files that we care about?